hkm/static/hkm/css/main.css

```diff
@@ -5890,102 +5875,174 @@ a.btn-delete-row {
-  .basket-footer {
-    background-color: rgba(255, 120, 101, 0.15); }
-  .basket-thumbnail img {
-    max-width: 150px;
-    height: auto;
-    border-radius: 2px;
-    box-shadow: 0 2px 12px -4px rgba(0, 0, 0, 0.5); }
+  margin-bottom: 15px; }
+  @media (min-width: 450px) {
+    .basket-row {
+      height: 300px;
+      grid-template-columns: 175px auto;
+      grid-template-rows: 30% 25% 25%; } }
+  @media (min-width: 850px) {
+    .basket-row {
+      height: 200px;
+      grid-template-columns: 175px 43% 43%;
+      grid-column-gap: 2%;
+      grid-template-rows: 50% 50%; } }
+  @media (min-width: 1050px) {
+    .basket-row {
+      grid-template-columns: 175px 46% 17% 15%;
+      grid-template-rows: 1fr;
+      grid-column-gap: 2%; } }
+
+.basket-thumbnail {
+  grid-row-start: 2; }
+  .basket-thumbnail_figure {
+    height: 125px; }
   .basket-thumbnail figcaption {
-    text-align: center;
-    margin-top: 12px;
+    margin-top: 5px;
     color: rgba(0, 0, 0, 0.5); }
-  .basket .discount {
-    padding-left: 30px; }
-  .basket-metadata {
-    align-self: flex-start;
-    text-align: center;
-    width: 420px;
-    margin-left: -3%; }
-    .basket-metadata p {
-      font-size: 16px; }
-  .basket-quantity {
-    display: flex;
-    align-items: center; }
-    .basket-quantity-value {
-      margin: 0; }
-    .basket-quantity a {
-      margin: 0 2em; }
-      .basket-quantity a svg.svg-icon {
-        width: 18px;
-        height: 18px; }
-    .basket-quantity a.btn-up svg.svg-icon {
-      transform: rotate(180deg); }
-  .basket-discounts-value {
-    font-weight: 700;
-    position: relative; }
-    .basket-discounts-value::after {
-      content: "€";
-      margin-left: 5px; }
-  .basket-discounts-total {
-    display: flex;
-    align-items: baseline;
-    justify-content: space-between;
-    width: 200px; }
-    .basket-discounts-total h5 {
-      letter-spacing: 0.6px;
-      font-size: 14px;
-      color: rgba(0, 0, 0, 0.5); }
-    .basket-discounts-total span {
-      color: #00C281;
-      font-size: 18px; }
-  .basket-price-item .basket-price-value {
-    font-size: 24px;
-    color: rgba(51, 51, 51, 0.8); }
-    .basket-price-item .basket-price-value::before {
-      content: "";
-      border-bottom: 3px solid rgba(255, 120, 101, 0.3);
-      position: absolute;
-      bottom: -7px;
-      left: 3px;
-      width: 97%; }
-  .basket-price-value {
-    font-weight: 700;
-    position: relative; }
-    .basket-price-value::after {
-      content: "€";
-      margin-left: 5px; }
-  .basket-price-total {
-    display: flex;
-    align-items: baseline;
-    justify-content: space-between; }
-    .basket-price-total h5 {
-      letter-spacing: 0.6px;
-      font-size: 18px;
-      color: rgba(0, 0, 0, 0.5); }
-    .basket-price-total span.basket-price-value {
-      color: #00C281;
-      font-size: 24px; }
-
-.btn-container {
-  margin-top: 2em; }
+  @media (min-width: 450px) {
+    .basket-thumbnail {
+      grid-row-start: 2;
```

Comment:
I usually find that doing placement with `column` and `row` is quite hard. I don't know what effect the changes are trying to achieve so this comment is something I am stating blind. It may well be that this is the best way. Looking at the PR, it seems that this pattern has been used a lot.

Reply:
I agree that is not the easiest thing. The order of the elements change quite a lot when screen size gets smaller, I just felt that doing it this way gave me a little more precision where the element should be in any given screen width.